unknown-carousel.html

```diff
@@ -284,19 +284,21 @@
               t = a[e],
                 "template" === t.localName || t.classList.contains("cloned") || h++;
           else
-            for (l = n.children,
-              o = 0,
-              r = l.length; o < r; o++)
-              t = l[o],
-                "template" !== t.localName && h++;
+            if(!!n) {
```

Comment:
```suggestion
            if(n) {
```
O próprio JS resolve isso pra você, aqui você terá perca de performance como discutimos e foi comparado pelo Will